crypto/ts/ts_verify_ctx.c

```diff
@@ -106,7 +105,6 @@ TS_VERIFY_CTX *TS_REQ_to_TS_VERIFY_CTX(TS_REQ *req, TS_VERIFY_CTX *ctx)
     ASN1_OCTET_STRING *msg;
     const ASN1_INTEGER *nonce;
 
-    OPENSSL_assert(req != NULL);
```

Comment:
While first omission is ok, because imminent crash is in next line. But this one might be meaningful to replace with plain assert. Just because reference to req is not in next line...